packages/graphql-server/src/resolvers/phApplication.ts

```diff
@@ -173,7 +173,8 @@ const listQuery = async (client: CustomResource<PhApplicationSpec, PhApplication
     phApplications.map(item => transform(item, kcAdminClient)));
 
   order = isEmpty(order) ? {displayName: 'asc'} : order;
-  return filter(transformedPhApplications, where, order);
+  const searchFields = ['id', 'appName', 'displayName'];
```

Comment:
`id`?

Reply:
Yes.. the app id. In the following screenshot, the id is `code-server-8xdqc`

![image](https://user-images.githubusercontent.com/860633/136179225-d97a7a68-cfc4-46a4-b998-03c5429b7ca1.png)